akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/TypedWatchingUntypedTest.java

```diff
@@ -84,11 +97,11 @@ public Receive createReceive() {
   public void testItWorks() {
     // #create
     ActorSystem as = ActorSystem.create();
-    ActorRef<Typed.Command> typed = Adapter.spawn(as, Typed.behavior(), "Typed");
+    ActorRef<Typed.Command> typed = Adapter.spawn(as, Typed.create(), "Typed");
     // #create
-    TestProbe probe = new TestProbe(as);
+    TestKit probe = new TestKit(as);
```

Comment:
Call it `testKit` rather than `probe` when it isn't a probe anymore.

Reply:
it was using classic scaladsl TestProbe from a Java test, which I found weird. Changed to classic javadsl, which doesn't have TestProbe but since we use it as a probe I think the name is good here.